src/main/java/org/example/objects/User.java

```diff
@@ -1,30 +1,39 @@
 package org.example.objects;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.example.fuzzy_matcher.domain.Document;
 import org.example.fuzzy_matcher.domain.Element;
 import org.example.fuzzy_matcher.domain.ElementType;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class User {
   private int id;
   private String name;
   private int age;
-  private List<String> favoriteGenres;
-  private List<String> favoriteArtists;
-  private List<String> favoriteSongs;
-  private List<String> recentlyPlayed;
-
-  public User(int id, String name, int age, List<String> favoriteGenres,
-              List<String> favoriteArtists, List<String> favoriteSongs,
-              List<String> recentlyPlayed) {
+  @JsonProperty("favorite_genres")
+  private List<String> favorite_genres;
+  @JsonProperty("favorite_artists")
+  private List<String> favorite_artists;
+  private List<String> favorite_songs;
+  private List<String> recently_played;
```

Comment:
We need to keep the camel case convention, please don't change the attribute syntax unless it is really necessary

Reply:
ok, the variables will not be changed